example/lib/util/ble_peripheral_category_stringifier.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:blemulator_example/model/ble_peripheral.dart';
+
+class BlePeripheralCategoryStringifier {
+  static String name(BlePeripheralCategory category) {
```

Comment:
This doesn't return the name, this parses a category for display purposes. If this is used only in one place, it can be moved to the view that calls it, if in multiple to some general util.

Reply:
Suggestion applied.